src/libs/elektra/keyset.c

```diff
@@ -709,6 +709,38 @@ ssize_t ksSearchInternal (const KeySet * ks, const Key * toAppend)
 	}
 }
 
+/**
```

Comment:
As discussed, ksSearchInternal should be static.

Reply:
done